spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -0,0 +1,299 @@
+require 'rails_helper'
+require 'action_account'
+require 'axe-rspec'
+
+RSpec.describe 'In Person Proofing Threatmetrix', js: true, allowed_extra_analytics: [:*] do
+  include InPersonHelper
+
+  let(:sp) { :oidc }
+  let(:user) { user_with_2fa }
+  let(:enrollment_code) do
+    JSON.parse(
+      UspsInPersonProofing::Mock::Fixtures.request_enroll_response,
+    )['enrollmentCode']
+  end
+  let(:stdout) { StringIO.new }
+  let(:stderr) { StringIO.new }
+  let(:argv) { ['review-pass', user.uuid, '--reason', 'INV1234'] }
+  let(:action_account) { ActionAccount.new(argv:, stdout:, stderr:) }
+  let(:review_pass) { ActionAccount::ReviewPass.new }
+  let(:review_reject) { ActionAccount::ReviewReject.new }
+  let(:include_missing) { true }
+  let(:config) { ScriptBase::Config.new(include_missing:, reason: 'INV1234') }
+  let(:enrollment) { InPersonEnrollment.last }
+  let(:profile) { enrollment.profile }
+
+  before do
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enforce_tmx).and_return(true)
+    ServiceProvider.find_by(issuer: service_provider_issuer(sp)).
+      update(in_person_proofing_enabled: true)
+  end
+
+  def deactivate_profile_update_enrollment(status:)
```

Comment:
So in thinking about this a bit more, this maybe isn't the best idea. I'm obscuring some important things that happen in the flow, and are basically stand-ins for the `get_usps_proofing_job`. Not sure what the best approach here is. I suppose this can be unwound so that the deactivation and updates are more explicit in the spec, or this thing can be moved to the `in_person_helper` if that is a more appropriate place for this sort of method.

Reply:
i do like the idea of moving this function out of the file...maybe to a threatmetrix_helper file? though the in_person_helper is also a good place

Reply:
those are great points. as for using that job I believe we could do that here. I'll try to update it

Reply:
I'm encountering an error related to usermailer when I try to include GetUspsProofingResultsJob. I wonder if that's why we're updating the enrollment as we are here, to get around the mailer issue?

Reply:
@svalexander that's exactly why I went with directly updating the enrollment.